.clang-format

```diff
@@ -5,12 +5,12 @@
 ## from http://releases.llvm.org/download.html
 ---
 # This configuration requires clang-format version 8.0.0 exactly.
-#  BasedOnStyle: Mozilla
+BasedOnStyle: Mozilla
 Language:        Cpp
 AccessModifierOffset: -2
-AlignAfterOpenBracket: Align
+AlignAfterOpenBracket: DontAlign
 AlignConsecutiveAssignments: false
-AlignConsecutiveDeclarations: true
+AlignConsecutiveDeclarations: false
```

Comment:
I strongly disagree with this change. Readability is paramount. We read the code hundreds of times more than when we write it. Readability was worth aligning the code before -- now we will have an automated way to keep the code readable. The code is much more important than the diffs.